components/TextFields/tests/unit/TextFieldTests.swift

```diff
@@ -52,11 +64,28 @@ class TextFieldTests: XCTestCase {
       XCTAssertEqual(textField.font, textFieldCopy.font)
       XCTAssertEqual(textField.hidesPlaceholderOnInput, textFieldCopy.hidesPlaceholderOnInput)
       XCTAssertEqual(textField.isEnabled, textFieldCopy.isEnabled)
+
+      if let leadingViewCopy = textField.leadingView as? UILabel {
```

Comment:
Yup! That's a typo. Which explains why it was passing. UILabel doesn't conform to copy. Pulling this out.